cves/cve_2021_41773.py

```diff
@@ -119,20 +156,23 @@ def validate(debug, container_name):
         if affected_version == constants.UNSUPPORTED:
             state[VULNERABILITY] = status_functions.not_determined(VULNERABILITY)
         elif affected_version:
-            permissions = filesystem_directory_configuration(debug, container_name)
-            if permissions == constants.UNSUPPORTED:
+            vulnerable_configuration = check_vulnerable_configuration(debug, container_name)
+            if vulnerable_configuration == constants.UNSUPPORTED:
                 state[VULNERABILITY] = status_functions.not_determined(VULNERABILITY)
-            elif permissions:
-                modules = apache_functions.loaded_modules('cgi_module', debug, container_name)
+            elif vulnerable_configuration:
+                cgi_module_name = 'LoadModule cgi_module modules/mod_cgi.so'
+                cgid_module_name = 'LoadModule cgid_module modules/mod_cgid.so'
```

Comment:
The only difference between the two lines is cgi -> cgid.
Can't we extract the line to a constant variable and use format string to create both alternatives?